spec/requests/users_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
```

Comment:
🚀 Well done!!! All the required tests are written and they all pass 👏🏾💯 

![image](https://github.com/HafizMassamTabraizKhan/Blog-app/assets/38088818/5d902805-c573-4ea1-8ffe-d9b9c4fb1736)